### PR TITLE
Ivdcm/gpb control

### PR DESCRIPTION
**Pull request by bonewell**
There are 2 channels. One of them is to transmit data and another is control channel.
Now control channel is only used to notify IVDCM about state of networks on SDL side.

### CODE REVIEW

File: src/appMain/smartDeviceLink.ini [hunk 1]
```diff
@@ -218,4 +218,5 @@ timeout_period_seconds = 10
 [IVDCM]
 ip = 127.0.0.1
 port = 5445
+control_port = 5446
```
Comment (DKlimenko) [inline src/appMain/smartDeviceLink.ini]: please add description for ports

File: src/components/ivdcm_module/src/gpb_data_sender_receiver.cc [hunk 2]
```diff
@@ -70,45 +70,134 @@ class GpbTransmitter : public threads::ThreadDelegate {
...
+  }
+  virtual void threadMain() {
+    LOG4CXX_AUTO_TRACE(logger_);
+    bool stop = false;
+    while (!stop) {
+      net::ConnectedSocket *conn = parent_->control_socket_->accept();
+      if (!conn) break;
+      parent_->controller_.set_socket(conn);
+      stop = parent_->controller_.MessageListenLoop(parent_);
+      parent_->controller_.set_socket(NULL);
```
Comment (DKlimenko) [inline src/components/ivdcm_module/src/gpb_data_sender_receiver.cc]: is it correct?

**Comment by DKlimenko** [comment]
reviewed

File: src/appMain/smartDeviceLink.ini (hunk 1 above)
Comment (bonewell) [inline src/appMain/smartDeviceLink.ini]: Done.

**Comment by VladSemenyuk** [comment]
Reviewed

**Pull request merged**